### PR TITLE
Microsoft.Build.Traversal: propogate well known global properties

### PR DESCRIPTION
**Pull request by forrestcoward**
See issue #207 for discussion on the change.

### CODE REVIEW

File: src/Traversal/Sdk/Sdk.targets [hunk 1]
```diff
@@ -127,6 +127,7 @@
     <MSBuild Projects="@(ProjectReference)"
              Condition="'%(ProjectReference.Build)' != 'false'"
              BuildInParallel="$([MSBuild]::ValueOrDefault('%(ProjectReference.BuildInParallel)', '$(BuildInParallel)'))"
+             Properties="%(ProjectReference.SetConfiguration); %(ProjectReference.SetPlatform); %(ProjectReference.SetTargetFramework)"
```
Comment (jeffkl) [inline src/Traversal/Sdk/Sdk.targets:130]: Change looks good, do you mind adding a unit test for each property to make sure it doesn't regress?

Comment (forrestcoward) [reply src/Traversal/Sdk/Sdk.targets:130]: Oh, yes!

**Comment by forrestcoward** [comment]
@jeffkl, do I need to update the version in `version.json` as well?

**Comment by jeffkl** [comment]
> do I need to update the version in version.json as well?

Yeah I suppose it changes enough behavior, might as well bump the minor version.  Thanks for suggesting that

File: src/Traversal/Sdk/Sdk.targets (hunk 1 above)
Comment (forrestcoward) [reply src/Traversal/Sdk/Sdk.targets:130]: I've added tests now. These fail before my change.

Comment (jeffkl) [reply src/Traversal/Sdk/Sdk.targets:130]: I'm looking into it...

**Review by jeffkl — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**